tetramer_validator/data/PTM_list.csv

```diff
@@ -0,0 +1,25 @@
+display_name,synonyms

```

Comment:
In general I prefer TSV. It's likely that some of these PTMs will contain commas, which requires quoting or escaping.

Reply:
Would it be better to store it in some sort of JSON format?

Reply:
If that makes the other code simpler, then fine. In general tables are easier to work with than JSON, so tables should be preferred for “flat” data.